cdap-data-fabric/src/main/java/io/cdap/cdap/spi/data/sql/PostgreSqlStructuredTableAdmin.java

```diff
@@ -186,6 +224,26 @@ private List<String> getCreateIndexStatements(StructuredTableId tableId, Set<Str
     return statements;
   }
 
+  private String getAddColumnStatement(StructuredTableId tableId,
```

Comment:
Annotate the method with `@Nullable` if it can return `null`. Also, consider using `Optional` instead of `null`.

Reply:
updated to Optional